change/@fluentui-react-compose-2020-06-26-11-31-26-xgao-compose-fix.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "resolveSlotProps: when slot/shorthand prop has JSX element as children, slot should be completely replace by it.",
```

Comment:
nit:
```suggestion
  "comment": "resolveSlotProps: Making change so that if slot/shorthand prop has JSX element as children, slot is completely replaced by it.",
```